plugins/inputs/wireless/wireless.go

```diff
@@ -0,0 +1,168 @@
+// +build linux
+
+package wireless
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+// default file paths
+const (
+	NET_WIRELESS = "/net/wireless"
+	NET_PROC     = "/proc"
+)
+
+// env variable names
+const (
+	ENV_WIRELESS = "PROC_NET_WIRELESS"
+	ENV_ROOT     = "PROC_ROOT"
+)
+
+var (
+	newLineByte = []byte("\n")
+)
+
+type wirelessInterface struct {
+	Interface string
+	Status    int64
+	Link      int64
+	Level     int64
+	Noise     int64
+	Nwid      int64
+	Crypt     int64
+	Frag      int64
+	Retry     int64
+	Misc      int64
+	Beacon    int64
+}
+
+// Wireless is used to store configuration values.
+type Wireless struct {
+	ProcNetWireless string `toml:"proc_net_wireless"`
+}
+
+var sampleConfig = `
+  ## file paths for proc files. If empty default paths will be used:
+  ##    /proc/net/wireless
+  proc_net_wireless = "/proc/net/wireless"
```

Comment:
Change this option to `# host_proc = "/proc"` and if set, use the environment variable `HOST_PROC`.  CHeck the bond input for an example.  This will make it work closer to the existing plugins including those that use gopsutil.